app/uploaders/image_uploader.rb

```diff
@@ -1,6 +1,9 @@
 class ImageUploader < CarrierWave::Uploader::Base
   include CarrierWave::MiniMagick
-  storage :fog
+
+  # Choose what kind of storage to use for this uploader:
+  # storage :file
+  storage :file
```

Comment:
Please make this conditional on whether the app is in development or production environment